docs/tutorials/data_management.md

```diff
@@ -86,7 +86,8 @@ fn main() {
 You might have noticed that the blob we generated is a hefty 13MB. This is no surprise, as we included `--all-keys` `--all-locales`. However, our binary only uses date formatting data in Japanese. There's room for optimization:
 
 ```console
-$ icu4x-datagen --cldr-tag latest --icuexport-tag latest --out my-data --format blob --keys-for-bin target/debug/myapp --locales ja
+$ cargo build
+$ icu4x-datagen --overwrite --cldr-tag latest --icuexport-tag latest --out my-data-blob --format blob --keys-for-bin target/debug/myapp --locales ja
```

Comment:
I've ommitted `cargo build` everywhere, if we're adding it we should also add  `target/debug/myapp`s invocations everywhere.

Reply:
I'll just remove it !